README.md

```diff
@@ -1,91 +1,189 @@
+of the wide variety of screen sizes. Originally, media queries could be used
+to set specific breakpoints based on a specific set of devices. As the market
+of web-connected electronics has grown, media queries have evolved to be more 
+inclusive of new sizes and aspect ratios. What are media queries, exactly, and 
+how can they be taken advantage of in building page layouts?
 
 ## Objectives
-1. The purpose of media queries.
-2. Media Query syntax.
-3. Accepted Devices.
-4. Min and max conditions.
-5. Choosing break point sizes.
 
-## Introducing Media Queries
-<iframe width="640" height="480" src="//www.youtube.com/embed/MYOT5FDG8gk?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe>
+1. What is the purpose of media queries
+2. What is the Media Query syntax
+3. Where should break points be created
+
+## What is the Purpose of Media Queries?
 
-*Note: Slides for this lecture video are provided in the resources at the bottom of this lesson.*
+Media queries are a feature of CSS. Media queries sets of styles that are 
+triggered by specific conditions. Media Queries are most used for measuring 
+the device viewport (screen) size and then adjust our CSS layout and content 
+accordingly so it best suits each size device. It can also be used for setting
+print styles. 
 
-### What are Media Queries
-Media Queries allow us to measure the device viewport (screen) size and then adjust our CSS layout and content accordingly so it best suits each size device.
+## What is the Media Query Syntax?
 
-### Syntax
 Below is the syntax for writing a media query statement within our CSS code:
+
 ```css
-@media [not|only] type [and] (expr) [,] … {
-    rules
+@media [not|only] mediatype [and] (expression) [,] … {
+  rules
 }
 ```
 
-We start by using the `@media` keyword followed by the condition statements that will trigger the media query on or off. The logical keywords "not" or "only" can optionally be provided to trigger queries for specific devices using `only` or for every device except a particular kind of device using the `not` keyword. These logical keywords are followed by the device type. At the time of writing this the only currently well supported device types are: screen, print, or all (meaning all devices). We can use `and` and `,` to separate conditions. Using `and` requires that both conditions on each side of the `and` are both true in order for the query to trigger on. The `,` stands for "or" meaning that only one of the conditions on either side of the comma has to be true for the query to trigger on. We use condition expressions surrounded by `()` parentheses to set up a condition that must be true in order for the query to trigger on and that will be false for the query to not be triggered. This is followed by a set of `{}` curly braces that enclose the CSS selectors and rules that will be applied when the media query is triggered.
+We start by using the `@media` keyword followed by conditional statements that
```

Comment:
Can you make this a little less scary for a newcomer here? Explain that you're about to provide examples?